### DISCUSSION TITLE
Lesson 11: problem with tokenCounter and  creating metadata_file_name video tag [11:11:15]

### DISCUSSION THREAD

**Discussion by santoroeth** [smartcontractkit/full-blockchain-solidity-course-py#1472] [Q&A]
Hi, I'm having problems with the "create_metada.py" file so I can't call "tokenCounter" and  the "for token_id" loop doesn't work, this is what the terminal prints.

![image](https://user-images.githubusercontent.com/97750911/167748570-614bb392-8301-472d-a8d6-4ec2914fbbd4.png)

Can someone give me a clue where I can be wrong? Thank you very much.

**Comment by cromewar** [comment]
Hello @santoroeth  remember you have to add plus 1 when you create the collectible on `fulfillRandomness` otherwise the counter will be always 0.
I hope this info might help.
